src/components/Form/FieldTemplates/ObjectFieldTemplate.tsx

```diff
@@ -1,4 +1,4 @@
-import Grid from '@mui/material/Grid';
+import { Grid } from '@mui/material';
```

Comment:
just in case, this component was created checking theirs and adding some sx rules. To keep consistency between Object templates fields I would keep this as default but not strong opinion on that.

here their component https://github.com/rjsf-team/react-jsonschema-form/blob/main/packages/mui/src/ObjectFieldTemplate/ObjectFieldTemplate.tsx